spec/unit/templates/domain_all_settings.xml

```diff
@@ -48,11 +48,13 @@
     </disk>
 
     <disk type='file' device='cdrom'>
+      <driver name='qemu' type='' />
```

Comment:
This doesn't seem correct, surely it's required to specify a valid type? Have a look at https://libvirt.org/formatdomain.html#elementsDisks, while it's valid to have `type` set to `qcow2` as you want to be able to, if specifying `type` it reads as though it needs to be a valid value. Making it default to `raw` as mentioned in the config code should be sufficient.